src/lava/magma/runtime/runtime_services/runtime_service.py

```diff
@@ -356,6 +356,7 @@ def run(self):
                         # Check if pause or stop received from Runtime
                         # TODO: Do we actualy need to wait for PMs to be in
                         # HOST or MGMT phase to stop or pause them?
+                        # PP: For pausing yes, otherwise get/set will not work
```

Comment:
Is there a better, probably code way of putting this statements in? These comments will deprecate fast and become confusing.

Reply:
Maybe file an issue for this discussion?

Reply:
It seems the TODO can be removed and replaced with a more descriptive comment that explains why we need to wait for PMs to be in HOST or MGMT phase.